src/bindings/python/src/pyopenvino/frontend/node_context.cpp

```diff
@@ -58,7 +58,6 @@ void regclass_frontend_NodeContext(py::module m) {
         [=](NodeContext& self, const std::string& name, const py::object& default_value, const py::object& dtype)
             -> py::object {
             auto any = self.get_attribute_as_any(name);
-            auto module = py::module_::import("openvino.runtime");
```

Comment:
Why did you remove it?

Reply:
unused variable, it was added in my ConversionExtension PR for some experiments